src/main/java/it/mulders/mcs/search/printer/CoordinatePrinter.java

```diff
@@ -31,4 +36,19 @@ private String first(final String... values) {
         }
         return null;
     }
+
+    private void printVulnerabilities(final ComponentReportResponse.ComponentReport componentReport,
+                                      final PrintStream stream) {
+        if (componentReport != null) { // will be null if --show-vulnerabilities cli arg is false
+            if (componentReport.vulnerabilities().length == 0) {
+                stream.println("No vulnerabilities found");
```

Comment:
```suggestion
                stream.println("No vulnerabilities reported");
```
Maybe some are found but haven't been reported yet... I don't want to make any false claims here :-)